Tests/SafeDIToolTests/SafeDIToolTests.swift

```diff
@@ -501,7 +501,7 @@ final class SafeDIToolTests: XCTestCase {
         )
     }
 
-    func test_run_writesConvenienceExtensionOnRootOfTree_whenRootInstantiatesPropertiesThatUtilizedSingleForwardedPropertyInSubBuilders() async throws {
```

Comment:
while I was here I realized that we were using the phrase `Utilized` when we meant `Utilizes` in our test names